crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -431,13 +431,25 @@ impl BladeRenderer {
     }
 
     pub fn update_drawable_size(&mut self, size: Size<DevicePixels>) {
+        self.update_drawable_size_impl(size, false);
+    }
+
+    /// Like `update_drawable_size` but skips the check that the size has changed. This is useful in
+    /// cases like restoring a window from minimization where the size is the same but the
+    /// renderer's swap chain needs to be recreated.
+    #[cfg_attr(any(target_os = "macos", target_os = "linux"), allow(dead_code))]
+    pub fn update_drawable_size_even_if_unchanged(&mut self, size: Size<DevicePixels>) {
+        self.update_drawable_size_impl(size, true);
+    }
+
+    fn update_drawable_size_impl(&mut self, size: Size<DevicePixels>, always_resize: bool) {
         let gpu_size = gpu::Extent {
             width: size.width.0 as u32,
             height: size.height.0 as u32,
             depth: 1,
         };
 
-        if gpu_size != self.surface_config.size {
+        if always_resize || gpu_size != self.surface_config.size {
             self.wait_for_gpu();
```

Comment:
perhaps we would instead do something like this:
```rust
if size.width.0 > 0 && size.height.0 > 0 {
  self.wait_for_gpu();
  self.gpu.resize(self.surface_config);
}
```
This will naturally make us to recreate the surface after the minimize is done, without `always_resize` flag

Reply:
Thanks for taking a look!  Wouldn't that cause this to always resize even if the size is the same as it as before?  (I assume that check is there for a good reason)

Reply:
After add log in handle_size_msg, I found the window size is (181, 24) in WM_SIZE event after minimize.

Reply:
> (181, 24)
this article might be old but it can be releated to this one
https://devblogs.microsoft.com/oldnewthing/20050210-00/?p=36483

Reply:
> > (181, 24)
 this article might be old but it can be releated to this one.
 https://devblogs.microsoft.com/oldnewthing/20050210-00/?p=36483